ui/partials/perf/comparechooserctrl.html

```diff
@@ -27,21 +27,23 @@
         <div class="panel-heading">New</div>
         <div class="panel-body">
           <label for="new-project-selector">Project</label>
-          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips()"/>
+          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips();getPreviousRevision()"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group" ng-class="{'has-error': newRevisionError}">
-            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision" placeholder="Select or enter a revision"/>
+            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision" placeholder="Select or enter a revision" ng-change="getPreviousRevision()"/>
```

Comment:
Hm well, the thing is that I added those so the search is started whenever we change the branch, or the revision (using either the list or typing in the field). I tried to set that in a low level function, e.g. getNewRevision or something, but I had issues. (I may give it another try though)